.github/workflows/ci.yml

```diff
@@ -29,13 +29,16 @@ jobs:
       - name: Build
         run: npm run build --if-present
 
+      - name: Lint
+        run: npm run lint
```

Comment:
@jerome-benoit @pioardi I have added the lint step here!

Reply:
This is the reason why I see warnings on this file indicating es-lint issues ? I would love that oh yeah

Reply:
Yes! This two lines provide you with warnings and errors in the `files changes` tab

Reply:
This definitevely remove any doubt for me about those changes, great !

Reply:
And then do you think that we should fix those warnings in this PR or into another one ?

Reply:
This PR is already huge, lets tackle these in another PR
Because they are in `examples` and `benchmarks`, maybe it would be nice if you can tackle them, so we can split the work a bit
But if you don't have time, I also could tackle these later on

Reply:
Sounds good but please do that as soon as we can so that when some PR comes we are able to see if there is any warning due to the changes